### PR TITLE
Image label

### PR DESCRIPTION
**Pull request by BenjaminCall**
**Description**

Enter a brief description of what you are accomplishing with this Pull Request.

### CODE REVIEW

File: client/js/_author/components/assessments/question_types/image_sequence/image_option.jsx [hunk 1]
```diff
@@ -35,31 +59,28 @@ export default function ImageOption(props) {
...
-        <label htmlFor="">Label</label>
-        <div className="au-c-input__contain">
-          <input className="au-c-text-input au-c-text-input--smaller" type="text" />
-          <div className="au-c-input__bottom" />
+        <div className="au-c-input au-c-input-label--left">
+          <label htmlFor="props.id">Label</label>
+          <div className="au-c-input__contain">
+            <input
+              defaultValue={this.props.labelText}
+              onChange={e => this.setState({ labelText: e.target.value })}
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/image_sequence/image_option.jsx]: This should be a controlled or uncontrolled component, not both.

Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/image_sequence/image_option.jsx]: Probably uncontrolled

Comment (BenjaminCall) [inline client/js/_author/components/assessments/question_types/image_sequence/image_option.jsx]: What do you mean by this?

File: client/js/middleware/serialization/qbank/deserializers/image_sequence.js [hunk 2]
```diff
@@ -8,18 +9,24 @@ function deserializeChoices(choices, correctAnswer, incorrectId) {
...
+
     newChoices[choice.id] = {
       id: choice.id,
       answerId: isCorrect ? correctAnswer.id : incorrectId,
-      text: choice.text,
+      text: image ? image.attr('src') : '',
+      altText: image ? image.attr('alt') : '',
       order: index + 1,
-      answerOrder: index,
+      answerOrder: isCorrect ? answerIndex : null,
```
Comment (nick-benoit14) [inline client/js/middleware/serialization/qbank/deserializers/image_sequence.js]: Are we still using answerOrder for anything? If not we could maybe just get rid of it.

File: client/js/middleware/serialization/qbank/serializers/image_sequence.js [hunk 3]
```diff
@@ -8,9 +8,13 @@ function serializeChoices(originalChoices, newChoiceAttributes) {
   const choices = _.map(originalChoices, (choice) => {
     const updateValues = newChoiceAttributes[choice.id];
     const newOrder = _.get(updateValues, 'order');
+    const labelText = _.get(updateValues, 'labelText', choice.labelText);
+    const imageSrc = choice.text || '';
```
Comment (nick-benoit14) [inline client/js/middleware/serialization/qbank/serializers/image_sequence.js]: Should we either use all `_.get` or `|| ''` for consistency sake?

Comment (BenjaminCall) [inline client/js/middleware/serialization/qbank/serializers/image_sequence.js]: meh..

File: client/js/middleware/serialization/qbank/deserializers/image_sequence.js (hunk 2 above)
Comment (BenjaminCall) [inline client/js/middleware/serialization/qbank/deserializers/image_sequence.js]: idk, are we?

**Pull request merged**